src/rule-search.ts

```diff
@@ -18,10 +18,19 @@ export class RuleInputProvider implements vscode.WebviewViewProvider {
 		this._extensionUri = _context.extensionUri;
 	}
 
-	public async toggle() {
+	public toggle() {
```

Comment:
Why this change?

Reply:
mostly realised it didn't need to be async